src/App.tsx

```diff
@@ -1,27 +1,17 @@
 import React from "react";
 
 import "./App.css";
-import logo from "./logo.svg";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import LandingPage from "./components/LandingPage";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-        <h1>Nurkec</h1>
-      </header>
-    </div>
+    <React.Fragment>
+      <Header />
+      <LandingPage />
+      <Footer />
```

Comment:
Instead of this I would rather have `<Layout>` component which will consist of `Header` and `Footer` and `children`.
This way you will avoid duplicating Header and Footer on other pages like Registration, Login, later toys page and other pages.

```
Layout.tsx
<>
  <Header />
  {children}
  <Footer />
</>